### PR TITLE
feat: add_angular_theming

### LINKED ISSUE #30: Feature: add Dark/White Mode theme to Web Frontend
As a developer,
I want to be able to get ready-to-use Dark/White Mode themes with Angular Materials theme feature.

https://material.angular.io/guide/theming
https://m2.material.io/design/color/dark-theme.html

### PR DESCRIPTION
**Pull request by Hubertgitck**
## Summary

Implemented Angular theming

## Details

Added white and dark theme and also applied those new themes to two components: button and expansion. There is a new class applied to <body> which is "class="mat-app-background"", that applies theming to a background of our application aswell. I did renamed styles.scss to theme.scss and moved it to "shared/styles".

## References

_Links which provides to external materials_ <br/>
[Angular Theming](https://material.angular.io/guide/theming#applying-a-theme-to-components) 

white
dark

### CODE REVIEW

File: binder-web-frontend/angular.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.scss"
+              "src/shared/styles/theme.scss"
```
Comment (StraykerPL) [inline binder-web-frontend/angular.json]: Make it `src/shared/styles`. Will be useful in the future.

**Review by StraykerPL — CHANGES_REQUESTED** [review]

**Comment by StraykerPL** [comment]
Fill missing PR's metadata: labels.

File: binder-web-frontend/angular.json (hunk 1 above)
Comment (Hubertgitck) [reply binder-web-frontend/angular.json]: Changed "src/shared/styles/theme.scss" -> "src/shared/styles.scss".

Comment (StraykerPL) [reply binder-web-frontend/angular.json]: I meant precisly `src/shared/styles` as folder, not file.

**Review by StraykerPL — CHANGES_REQUESTED** [review]
Reopened previous comment.

File: binder-web-frontend/angular.json (hunk 1 above)
Comment (Hubertgitck) [reply binder-web-frontend/angular.json]: i made styles.scss global styles file where we will import all the other styles and left only "src/shared/styles/styles.scss" in angular.json

File: binder-web-frontend/src/shared/styles/styles.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+@import 'theme.scss';
```
Comment (StraykerPL) [inline binder-web-frontend/src/shared/styles/styles.scss]: `use` instruction here. And no need to use file extension here.

**Review by StraykerPL — CHANGES_REQUESTED** [review]

File: binder-web-frontend/src/shared/styles/styles.scss (hunk 2 above)
Comment (Hubertgitck) [reply binder-web-frontend/src/shared/styles/styles.scss]: changed to @use

**Review by StraykerPL — APPROVED** [review]

**Pull request merged**